tests/src/python/test_qgsgeometrygeneratorsymbollayer.py

```diff
@@ -463,6 +466,31 @@ def test_geometry_function(self):
             )
         )
 
+    def test_field_geometry(self):
+        """
+        Use a geometry field
+        """
+
+        points = QgsVectorLayer('Point?crs=epsg:2154&field=other_geom:geometry(0,0)', 'Points', 'memory')
+        f = QgsVectorLayerUtils.createFeature(points,
+                                              QgsGeometry.fromWkt('Point(5 4)'),
+                                              {0: QgsReferencedGeometry(QgsGeometry.fromWkt('LineString(5 6, 7 8)'), QgsCoordinateReferenceSystem("EPSG:4326"))})
+        points.dataProvider().addFeature(f)
+        other_layer = QgsGeometryGeneratorSymbolLayer.create({'geometryModifier': '"other_geom"', 'outline_color': 'black', 'SymbolType': 'Line'})
+        points.renderer().symbol().changeSymbolLayer(0, other_layer)
```

Comment:
Can you set the subSymbol here to a much fatter line symbol? The current reference image will be too sensitive to cross-platform anti-aliasing differences.